daml-assistant/daml-helper/src/DA/Daml/Helper/Ledger.hs

```diff
@@ -216,3 +236,79 @@ runLedgerNavigator flags remainingArguments = do
           ]
         , ["  }"]
         ]
+
+----------------
+-- HTTP JSON API
+----------------
+
+newtype Method = Method
+  { unMethod :: BS.ByteString
+  } deriving IsString
+
+newtype Path = Path
+  { unPath :: BS.ByteString
+  } deriving IsString
```

Comment:
There are also HTTP APIs with more types like `req`. Maybe worth considering to use one if you care about this.

Reply:
@cocreature could you point me to one?

Reply:
It’s literally called [req](https://hackage.haskell.org/package/req)

Reply:
Ah, was getting your comment wrong. I think for this simple use case using plain http-client-simple should be good enough.

Reply:
agreed